llvm/lib/LTO/LTO.cpp

```diff
@@ -1634,8 +1639,6 @@ class WriteIndexesThinBackend : public ThinBackendProc {
       const std::map<GlobalValue::GUID, GlobalValue::LinkageTypes> &ResolvedODR,
       MapVector<StringRef, BitcodeModule> &ModuleMap) override {
     StringRef ModulePath = BM.getModuleIdentifier();
-    std::string NewModulePath =
-        getThinLTOOutputFile(ModulePath, OldPrefix, NewPrefix);
 
     if (LinkedObjectsFile) {
```

Comment:
Can you add a comment about why this is not in the parallel section below

Reply:
I think it is

Reply:
Confused - I thought the LinkedObjectsFile writing is explicitly done here serially?

Reply:
Oh sorry, I thought you were referring to the removed `NewModulePath` lines only. Yes I can add a comment about why the `LinkedObjectsFile` business needs to be done serially.